### PR TITLE
Changed print statements and input statements

### PR DESCRIPTION
**Pull request by sulavmhrzn**
Changed print and input statements in NEA.py file

### CODE REVIEW

**Review by ashiishme — COMMENTED** [review]
@hemantapokharel @TheBinitGhimire @pr0d33p 

Please check the refactorization and leave your review before merge.

File: NEA.py [hunk 1]
```diff
@@ -14,96 +14,105 @@
...
 
-	else:
-		print add + " Not found!"
-		exit(1)
+    else:
+        print(add + " Not found!")
+        exit(1)
 
-print  ' _____________________________________' 
+print ' _____________________________________'
```
Comment (hemantapokharel) [inline NEA.py:69]:
```suggestion
print(' _____________________________________')
```

File: NEA.py [hunk 2]
```diff
@@ -14,96 +14,105 @@
...
 
 file = 'address.txt'
 
+
 def exit2():
 
-	print('\n\033[94m [+] Our Site: www.askbuddie.com\n [+] Join Us: www.fb.com/groups/askbuddie\n [+] Our FB page: www.fbcom/askbuddie\n [+] ABOS: https://github.com/askbuddie')
-	print('\n\033[93m Thanks for using NEA Bill Checker...\n\n')
-	exit()
+    print('\n\033[94m [+] Our Site: www.askbuddie.com\n [+] Join Us: www.fb.com/groups/askbuddie\n [+] Our FB page: www.fbcom/askbuddie\n [+] ABOS: https://github.com/askbuddie')
```
Comment (hemantapokharel) [inline NEA.py:46]:
```suggestion
    print('\n\033[94m [+] Our Site: www.askbuddie.com\n [+] Join Us: www.fb.com/groups/askbuddie\n [+] Our FB page: www.fb.com/askbuddie\n [+] ABOS: https://github.com/askbuddie')
```

File: NEA.py [hunk 3]
```diff
@@ -14,96 +14,105 @@
...
+    print('\n\033[93m Thanks for using NEA Bill Checker...\n\n')
+    exit()
+
```
Comment (hemantapokharel) [inline NEA.py:50]: Error handling while reading address.txt.
```suggestion
try:
    open(file)
except Exception:
    print(f"\n\033[91m [-] {file} not found.")
    exit2()
```

**Comment by prdp1137** [comment]
This is currently be modified for use of address.txt in another way and I'm also taking this to python 3 at the same time. But thanks for PR.

**Comment by prdp1137** [comment]
I have modified and made simpler using bs4, so, I suggest you to help us contributing there. Thanks, for the PR. I'm closing this as of now.

**Pull request closed without merging**